lang/stata/stata.py

```diff
@@ -0,0 +1,170 @@
+from talon import Context, actions, settings
+
+ctx = Context()
+
+ctx.matches = r"""
+code.language: stata
+"""
+
+# functions.py
+ctx.lists["user.code_parameter_name"] = {
+    # regressions
+    "v c e cluster": "vce(cluster)",
```

Comment:
I could be wrong, but I think usually for single letters we usually use capitals, although I'm not sure it actually matters. Can just ignore this suggestion if it works ok as is.